console/src/components/Services/ApiExplorer/Actions.js

```diff
@@ -44,6 +44,20 @@ const CREATE_WEBSOCKET_CLIENT = 'ApiExplorer/CREATE_WEBSOCKET_CLIENT';
 const FOCUS_ROLE_HEADER = 'ApiExplorer/FOCUS_ROLE_HEADER';
 const UNFOCUS_ROLE_HEADER = 'ApiExplorer/UNFOCUS_ROLE_HEADER';
 
+let WEBSOCKET_SUBSCRIPTION_CLIENT;
+
+const getSubscriptionInstance = (url, headers) => {
+  return new SubscriptionClient(url, {
+    connectionParams: {
+      headers: {
+        ...headers,
+      },
+      lazy: true,
+    },
+    reconnect: true,
+  });
+};
+
```

Comment:
Might seem a little over the top...will remove it if that is the case. Did this only because it makes the changes below more readable.